sklearn/linear_model/_least_angle.py

```diff
@@ -822,7 +822,7 @@ class Lars(MultiOutputMixin, RegressorMixin, LinearModel):
         ``n_alphas`` is either ``n_nonzero_coefs`` or ``n_features``, \
         whichever is smaller.
 
-    active_ : list, length = n_alphas | list of n_targets such lists
+    active_ : list of length (n_alphas) or list of length (n_targets)
```

Comment:
I think it's good to keep the info that when n_targets > 1, it's a list of lists of length n_alphas

```suggestion
    active_ : list of length n_alphas or list of n_targets such lists
```

Reply:
@jeremiedbb shouldn't it be at least `list of n_targets such lists` ?

Reply:
@adrinjalali what do you mean ?